src/content/community/releases/_index.en.md

```diff
@@ -4,6 +4,15 @@ title = "Releases"
 weight = 40
 +++
 <!-- markdownlint-disable no-duplicate-header -->
+## v0.13.2
+
+* Added support for OpenShift 4.12.
+* Service Discovery now returns a DNS error message in the response body when no matching records are found for the query to
```

Comment:
```suggestion
* Service Discovery now returns a DNS error message in the response body when no matching records are found when queried about
```

Reply:
We already have this note for 0.14.0,  should I change there as well?

Reply:
Yes, let's fix both for consistency.